apps/api/package.json

```diff
@@ -17,7 +17,7 @@
     "check-types": "tsc --noEmit",
     "deploy": "wrangler deploy --minify",
     "dev": "wrangler dev",
-    "postinstall": "bun run cf-typegen"
+    "postinstall": "[ -f .env.example ] && [ ! -f .env ] && cp .env.example .env || true; bun run cf-typegen"
```

Comment:
I'd love to see the `-v` option for the `cp` command so we get the visual indicator that the file get copied.